### PR TITLE
Add ObjC version of AnyJSON to message

### PR DESCRIPTION
**Pull request by wkal-pubnub**

### CODE REVIEW

File: Sources/PubNub/KMM/Wrappers/AnyJSONObjC.swift [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+@objc
+public class AnyJSONObjC : NSObject {
+    let value: AnyJSON
+    public init(_ value: AnyJSON) {
+        self.value = value
+    }
+    
+    @objc
+    public func asString() -> String? {
+        return value.stringOptional
```
Comment (jguz-pubnub) [inline Sources/PubNub/KMM/Wrappers/AnyJSONObjC.swift:22]: You can skip the `return` keyword for single-line expression functions that returns a value

Comment (wkal-pubnub) [reply Sources/PubNub/KMM/Wrappers/AnyJSONObjC.swift]: done

File: Sources/PubNub/KMM/Wrappers/PubNubMessageObjC.swift [hunk 2]
```diff
@@ -13,24 +13,28 @@ import Foundation
 @objc
 public class PubNubMessageObjC: NSObject {
   init(message: PubNubMessage) {
-    self.payload = message.payload
+    self.payload = AnyJSONObjC(message.payload as! AnyJSON)
```
Comment (jguz-pubnub) [inline Sources/PubNub/KMM/Wrappers/PubNubMessageObjC.swift:16]: You can replace force casting with `AnyJSONObjC(message.payload.codableValue)`

File: Sources/PubNub/KMM/Wrappers/PubNubMessageObjC.swift [hunk 3]
```diff
@@ -13,24 +13,28 @@ import Foundation
...
     self.actions = message.actions.map { PubNubMessageActionObjC(action: $0) }
     self.publisher = message.publisher
     self.channel = message.channel
     self.subscription = message.subscription
     self.published = message.published
-    self.metadata = message.metadata
+    self.metadata = if let metadata = message.metadata {
+        AnyJSONObjC(metadata as! AnyJSON)
```
Comment (jguz-pubnub) [inline Sources/PubNub/KMM/Wrappers/PubNubMessageObjC.swift:23]:
```suggestion
        AnyJSONObjC(metadata.codableValue)
```

File: Sources/PubNub/KMM/Wrappers/PubNubMessageObjC.swift (hunk 2 above)
Comment (jguz-pubnub) [inline Sources/PubNub/KMM/Wrappers/PubNubMessageObjC.swift:16]:
```suggestion
    self.payload = AnyJSONObjC(message.payload.codableValue)
```

File: Sources/PubNub/KMM/Wrappers/PubNubMessageObjC.swift (hunk 3 above)
Comment (wkal-pubnub) [reply Sources/PubNub/KMM/Wrappers/PubNubMessageObjC.swift]: done

File: Sources/PubNub/KMM/Wrappers/PubNubMessageObjC.swift (hunk 2 above)
Comment (wkal-pubnub) [reply Sources/PubNub/KMM/Wrappers/PubNubMessageObjC.swift]: done

**Review by jguz-pubnub — APPROVED** [review]

**Pull request merged**